src/main/java/com/joyent/manta/monitor/MantaMonitorJerseyServer.java

```diff
@@ -141,10 +220,35 @@ private void configureServer() {
             httpsConfig.addCustomizer(src);
 
             SslContextFactory sslContextFactory = new SslContextFactory();
-            sslContextFactory.setKeyStorePath(System.getenv("KEYSTORE_PATH"));
+
+            String userHomePath = System.getProperty("user.home");
+
+            URI keystoreURI = getURIFromString(System.getenv("KEYSTORE_PATH"));
```

Comment:
Are we reading the private TLS key from Manta? Would this be a good idea from a security perspective? Is this feature needed?

Reply:
This is not the key but the keystore, that is created using the key (and the certificate). So the idea is that the user will create the keystore file in (PKCS12 format) on his local machine, then upload it in the manta store and then while running the app, using docker run, will provide the path to the file and also the password needed to access the keystore. Without the password, the keystore (or the truststore) will be inaccessible.

Reply:
:+1: